docs/5g-on-helium/mobile-poc.mdx

```diff
@@ -8,46 +8,76 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
 Hotspots are located where they claim. Put another way, PoC tries to verify, on an ongoing basis,
 that Hotspots are honestly representing their location and the wireless network coverage they are
 creating from that location.
 
 LoRaWAN Hotspots utilize a PoC method that benefits from Hotspots being used as senders ("Beacon")
-and receivers ("Witness") of PoC activity, read more here:
+and receivers ("Witness") of PoC activity; read more here:
 [Proof of Coverage](/blockchain/proof-of-coverage)
 
-While all current 5G Hotspot models _also_ participate in the LoRaWAN network and PoC, the CBRS
+While all current 5G Hotspot models _also_ participate in the LoRaWAN Network and PoC, the CBRS
 Small Cell and 5G aspects of a 5G Hotspot require a new approach to validating PoC.
 
 :::info **One Way Traffic**
 
-CBRS Small Cell Radios can only operate as transmitters, they are not able to receive incoming data
-and thus can not "hear" one another directly.
+CBRS Small Cell Radios can only operate as transmitters; they cannot receive incoming data and thus
+can not "hear" one another directly.
 
 :::
 
-MOBILE Rewards are earned as one "lump sum" by a Wallet with a 5G Hotspot with at least one
-connected, SAS registered, CBRS Small Cell Radio every roughly 24-hour interval. This allows a high
-tolerance for downtime in infrastructure that is beyond the 5G Hotspot Owner's control.
+---
+
+## How To Mine MOBILE
+
+Several factors contribute to the amount of MOBILE a 5G Hotspot will earn over a given period. This
+page covers some higher-level concepts, design elements, and rules to keep in mind that will help
+better explain what you might earn and why. This section covers these and some mining and token
+reward basics.
+
+The Helium Blockchain rewards 5G Hotspots with CBRS Radios for providing 5G wireless coverage and
+verifying the Helium Network with the Helium MOBILE Token.
+
+Mining MOBILE and providing 5G coverage requires a 5G Hotspot and one or more compatible CBRS Small
+Cell Radios. These components can be purchased as a bundle or separately. All 5G Hotspots must be
+online and registered in the SAS (Spectrum Access System) maintained by the FCC to earn MOBILE
+rewards. Once properly deployed, you, as the owner, are not required to do anything else with your
+5G Hotspot to earn MOBILE.
+
+More than one Small Cell Radio can be connected to a single 5G Hotspot to create a larger 5G
+coverage area. 5G Hotspots can support as many Small Cell Radios as would saturate their bandwidth.
+
+Connecting more than one Small Cell Radio will require an additional networking switch. Refer to the
+5G Hotspot Maker installation guides for specific instructions.
+
+:::info
+
+During [Genesis](./mobile-poc#mobile-genesis), 5G Hotspots earn MOBILE by reporting their status via
+a "heartbeat," an automatic reporting process to the Helium Blockchain that the Small Cell Radio is
+online and registered with the SAS.
```

Comment:
Maybe something like "More coverage metrics are being continuously added during Genesis period to make sure the 5G Network is reliable and of a high quality."